cpu/esp32/Makefile.include

```diff
@@ -51,6 +51,14 @@ ifneq (,$(filter esp_wifi,$(USEMODULE)))
     USEMODULE += esp_wifi_any
 endif
 
+ifneq (, $(filter spiffs, $(USEMODULE)))
+    export RIOT_TEST_TIMEOUT = 300
```

Comment:
```suggestion
  export RIOT_TEST_TIMEOUT = 300
```

Reply:
I'm really sorry, this is true for most of my makefiles. I didn't know that convention when I was starting with RIOT.  At least I didn't find any clue in the encoding convention and my editor is configured with 4 spaces indent because of the coding conventions in RIOT.

Should I correct it in the compete file with this PR to be consistent or just this one?

BTW, there are a number of other makefiles having the same problem, even `cpu/native/Makefile.include`.

Reply:
I only fixed this one.

Reply:
> Should I correct it in the compete file with this PR to be consistent or just this one?

If you could provide a PR that fixes that it would be nice. Also, it would be great to move the uses of USEMODULE/FEATURES_REQUIRED in `Makefile.dep` and keep other configuration setting in Makefile.include.
All this could be done in a single PR. I can review if you open such a PR.

Reply:
I've always had problems with what exactly should be in `Makefile.include` and what in `Makefile.dep`.

Should there be only `USEMODULE` and `FEATURES_REQUIRED` settings in `Makefile.dep`, or could there also be settings for` CFLAGS`, `INCLUDES`,` LINKFLAGS` ... Or should the latter all be in `Makefile.include`?

I had also problems with the fact that the `Makefile.dep` is included as the last file, but some module dependencies have to be already known in `Makefile.include`. That's why there are some module dependency settings in `Makefile.include`.